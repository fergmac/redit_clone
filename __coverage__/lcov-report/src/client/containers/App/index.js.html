<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for src/client/containers/App/index.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../../prettify.css" />
    <link rel="stylesheet" href="../../../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../../../index.html">All files</a> / <a href="index.html">src/client/containers/App</a> index.js
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">0% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>0/34</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">0% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>0/29</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">0% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>0/11</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">0% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>0/7</span>
      </div>
    </div>
  </div>
  <div class='status-line low'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32</td><td class="line-coverage quiet"><span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">import React, { PureComponent } from 'r<span class="cstat-no" title="statement not covered" >eact';<span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="fstat-no" title="function not covered" ><span class="fstat-no" title="function not covered" ><span class="fstat-no" title="function not covered" ></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span>
import Artists from '../../components/Artists';
import styles from './styles.css';
import { connect } from 'react-redux';
import { fetchGenres } from '../../../shared/redux/modules/genres';<span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="fstat-no" title="function not covered" ><span class="fstat-no" title="function not covered" ><span class="fstat-no" title="function not covered" ></span></span></span></span></span></span></span></span></span></span></span></span></span>
&nbsp;
class <span class="cstat-no" title="statement not covered" ><span class="fstat-no" title="function not covered" >Ap</span>p extends PureComponent {<span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="fstat-no" title="function not covered" ><span class="fstat-no" title="function not covered" ></span></span></span></span></span></span></span>
&nbsp;
  componentDidMount() {
    th<span class="cstat-no" title="statement not covered" >is.props.fetchGenres();</span>
  }<span class="fstat-no" title="function not covered" ></span>
&nbsp;
  render() {
    re<span class="cstat-no" title="statement not covered" >turn (</span>
      &lt;div&gt;
        &lt;Artists /&gt;
        &lt;div className={styles.viewContainer}&gt;
          {this.props.children}
        &lt;/div&gt;
      &lt;/div&gt;
    );
  }<span class="cstat-no" title="statement not covered" ></span>
}
&nbsp;
const mapDispatchToProps <span class="cstat-no" title="statement not covered" >= (dispat<span class="fstat-no" title="function not covered" >ch) =&gt; ({</span></span>
  fetchGenres: () =&gt; {
    dispatch(fetchGenres());
  },
});
&nbsp;
export default connect(null, mapDispatchToProps)(App);
&nbsp;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="http://istanbul-js.org/" target="_blank">istanbul</a> at Wed Nov 09 2016 16:25:23 GMT-0800 (PST)
</div>
</div>
<script src="../../../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../../../sorter.js"></script>
</body>
</html>
