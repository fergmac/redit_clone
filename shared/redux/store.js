import { createStore } from 'redux';
import reducers from './reducers';

const store = createStore(reducers, {});

console.log(store);

export default store;
