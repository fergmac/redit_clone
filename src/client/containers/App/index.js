import React, { PureComponent } from 'react';
import Artists from '../../components/Artists';
// import styles from './styles.css';
import { connect } from 'react-redux';
import { fetchGenres } from '../../../shared/redux/modules/genres';

const styles = {
  viewContainer: {
    fontFamily: 'Roboto, sans-serif',
    minHeight: '100vh',
    boxSizing: 'border-box',
    display: 'flex',
    flexDirection: 'column',
    div: {
      flex: '1',
    },
  },
};

class App extends PureComponent {

  componentDidMount() {
    this.props.fetchGenres();
  }

  render() {
    return (
      <div>
        <Artists />
        <div className={styles.viewContainer}>
          {this.props.children}
        </div>
      </div>
    );
  }
}

const mapDispatchToProps = (dispatch) => ({
  fetchGenres: () => {
    dispatch(fetchGenres());
  },
});

export default connect(null, mapDispatchToProps)(App);
