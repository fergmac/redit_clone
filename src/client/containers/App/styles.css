/*.viewContainer {
	font-family: Roboto, sans-serif;
	min-height: 100vh;
  box-sizing: border-box;
	display:flex;
	flex-direction: column;
}

.viewContainer > div {
	flex: 1;
}*/
