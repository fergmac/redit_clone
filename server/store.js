import { createStore } from 'redux';
import reducers from '../src/shared/redux/reducers';

export default createStore(reducers, {});


