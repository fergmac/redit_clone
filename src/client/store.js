import { createStore, applyMiddleware } from 'redux';
import createLogger from 'redux-logger';
import reduxThunk from 'redux-thunk';
import reducers from '../shared/redux/reducers';
import remoteMiddleware from './socket/remoteMiddleware';
import socket from './socket';

const logger = createLogger();

const middlewares = applyMiddleware(reduxThunk, remoteMiddleware(socket), logger);

const store = createStore(reducers, middlewares);

export default store;
