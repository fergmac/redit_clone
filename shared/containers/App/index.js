import React from 'react';
import Categories from '../../components/Categories';
import styles from './styles.css';


const App = (props) => (
  <div>
    <Categories />
    <div className={styles.viewContainer}>
      {props.children}
    </div>
  </div>
);

App.propTypes = {
  children: React.PropTypes.object,
};

export default App;

