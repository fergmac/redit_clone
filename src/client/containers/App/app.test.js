/* eslint-disable */
import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import App from './index';
import { Provider } from 'react-redux';
import store from '../../store';


describe('AlbumList', () => {
    const children = {
        
    }
    it('renders App', () => {
       const wrapper = shallow(
           <Provider store={store}>
                <App 
                    
                />
          </Provider>
       )
        expect(shallowToJson(wrapper)).toMatchSnapshot()
    })
})