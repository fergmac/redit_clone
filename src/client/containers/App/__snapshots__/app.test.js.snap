exports[`AlbumList renders App 1`] = `<Connect(App) />`;
